### PR TITLE
feat(preview): additional content in preview container

### PR DESCRIPTION
**Pull request by PeterDraex**
## What kind of change does this pull request introduce?

New feature

## What is the new behavior?

Children of `<SandpackPreview>` are rendered at the end of `div.sp-preview-container`.

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

Used in a documentation site.

## Checklist

- [x] Documentation;
- [x] Storybook (if applicable);
- [ ] Tests;
- [x] Ready to be merged

### CODE REVIEW

File: sandpack-react/src/components/Preview/Preview.stories.tsx [hunk 1]
```diff
@@ -122,3 +122,14 @@ export const AdditionalButtons: React.FC = () => (
     </SandpackLayout>
   </SandpackProvider>
 );
+
+export const AdditionalContent: React.FC = () => (
```
Comment (danilowoz) [inline sandpack-react/src/components/Preview/Preview.stories.tsx:126]: Awesome, I appreciate your effort to keep the documentation and storybook up to date!

File: sandpack-react/src/components/Preview/index.tsx [hunk 2]
```diff
@@ -34,6 +34,7 @@ export interface PreviewProps {
   showRefreshButton?: boolean;
   showSandpackErrorOverlay?: boolean;
   actionsChildren?: JSX.Element;
+  children?: JSX.Element;
```
Comment (danilowoz) [inline sandpack-react/src/components/Preview/index.tsx:37]: Let's use `React.FC instead, in order to keep the same format for all component with children

File: website/docs/docs/advanced-usage/components.md [hunk 3]
```diff
@@ -150,6 +150,9 @@ The `<SandpackPreview />` component also allows you to add additional buttons to
   </SandpackLayout>
 </SandpackProvider>
 
+### Additional content
+For advanced use cases, children of `<SandpackPreview>` are rendered at the end of the [preview container](https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/components/Preview/index.tsx#L117).
```
Comment (danilowoz) [inline website/docs/docs/advanced-usage/components.md:154]: This link may become outdated over time, so I'd leave the content as you have now, but without the link.

**Comment by PeterDraex** [comment]
Ready to merge in my opinion

**Pull request merged**